include/wx/event.h

```diff
@@ -3044,6 +3046,32 @@ class WXDLLIMPEXP_CORE wxDisplayChangedEvent : public wxEvent
     virtual wxEvent *Clone() const wxOVERRIDE { return new wxDisplayChangedEvent(*this); }
 };
 
+/*
+ wxEVT_DPI_CHANGED
+ */
+class WXDLLIMPEXP_CORE wxDPIChangedEvent : public wxEvent
+{
+public:
+    explicit
+    wxDPIChangedEvent(const wxSize& oldDPI = wxDefaultSize,
+                      const wxSize& newDPI = wxDefaultSize)
+        : wxEvent(0, wxEVT_DPI_CHANGED)
+        , m_oldDPI(oldDPI)
```

Comment:
Style nitpick: could you please use trailing commas. I'm aware of the benefits of this style but, as often, I think consistency is more important.